### ISSUE TITLE
DropdownV2 has multiple tabbable element

### ISSUE THREAD

**Issue by nicoxxie** [IBM/carbon-components-react#1878]
## Environment

> Operating system
MACOSX

> Browser
Chrome

> Automated testing tool and ruleset
DAP

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?
4.0.10

> What did you expect to happen?
Only one tabbable element on the main dom

> What happened instead?
two tabblable element

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox
Directly viewable on http://react.carbondesignsystem.com/?selectedKind=DropdownV2&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel
## Additional information

- Screenshots or code
- Notes

![81d65928-f70d-11e8-896d-708051638e54](https://user-images.githubusercontent.com/17915105/52810641-6066df80-30d6-11e9-817a-eaeb591b8b9f.png)
![f7c2486a-2087-11e9-9ccb-caba4c2bdb06](https://user-images.githubusercontent.com/17915105/52810643-6066df80-30d6-11e9-9fea-442c48c780ed.png)

**Comment by dakahn** [comment]
Too right! Thanks for submitting the issue. Currently working on this error rn. I'll link this issue in the subsequent PR. 👍🏽

**Comment by dakahn** [comment]
Doing a round of testing and it looks like this issue is resolved. 👍 Happy to reopen if I need to though.

**Comment by nicoxxie** [comment]
@dakahn I'm still seeing two tabbale item for Dropdown.  Think this should be removed from the listbox?

![Screen Shot 2019-03-27 at 3 22 41 PM](https://user-images.githubusercontent.com/17915105/55109726-7a67ea80-50a4-11e9-9884-ca1cbcba24e1.png)

**Comment by dakahn** [comment]
Mega weird DAP doesn't catch that anymore. I'll push up a fix right quick 👍🏽

**Comment by emyarod** [comment]
@nicoxxie can you verify if this issue still exists? I think I resolved it in https://github.com/IBM/carbon-components-react/pull/2158

**Comment by eason123deng** [comment]
Hi @emyarod @dakahn, I'm a dev from SPSS statistics, we migrated to Carbon recently (our version is 6.103.1) and it seems like we need to tab twice + down arrow key before opening the dropdown. Since SPSS statistics requires full accessibility before we can release the product, if this is not already addressed can we prioritize addressing this issue? 

Thanks

**Comment by nicoxxie** [comment]
@dakahn @emyarod Sorry I was on vacation, unfortunately we have not migrated to carbon v10 yet... and I see this still happens on the latest v9 carbon-components-react 6.114.1, will see if the update happens soon.

**Comment by emyarod** [comment]
@eason123deng @nicoxxie got it, will have to look into backporting my fix from v10
